src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -125,10 +128,50 @@ public Response isAdmin(@PathParam("instanceName") String instanceName) throws I
         return Response.ok(body).build();
     }
 
+    @GET
+    @Path("{instanceName}/admin/list")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getAdminLis(@PathParam("instanceName") String instanceName) throws IOException, ApiException, GeneralSecurityException {
+        UserProfile userProfile = UserProfileManager.getUserProfile();
+        String token = userProfile.getAccessToken();
+        GitHubClient client = new GitHubClient();
+        client.setOAuth2Token(token);
+
+        Kabanero instance = KabaneroClient.getAnInstance(instanceName);
+        if (instance == null) {
+            return Response.status(404).entity(new ResponseMessage(instanceName + " not found")).build();
+        }
+
+        String instanceGithubOrg = instance.getSpec().getGithub().getOrganization();
+        TeamService teamService = new TeamService(client);
+
+        Map<Team, List<User>> map = new HashMap<Team, List<User>>();
+        List<User> instanceAdmins = new ArrayList<>();
+        Map<String, Object> body = new HashMap<>();
+
+        for (Team orgTeam : teamService.getTeams(instanceGithubOrg)) {
+            List<User> orgTeamMembers = new ArrayList<>();
+
+            for (User instanceTeamMember : teamService.getMembers(orgTeam.getId())) {
+                orgTeamMembers.add(new UserService(client).getUser(instanceTeamMember.getLogin()));
+                map.put(orgTeam, orgTeamMembers);
```

Comment:
I feel like this should be outside the inner loop? Only do 1 put for each orgTeam.